postgres/CHANGELOG.md

```diff
@@ -1,18 +1,14 @@
 # CHANGELOG - postgres
 
-1.0.1/2017-04-24
-==================
+## 1.0.1 / Unreleased
 
 ### Changes
 
-* [DEPENDENCY] bump psycopg2 to 2.7.1. See [#295][]
+* [IMPROVEMENT] provide a meaningful error when custom metrics are misconfigured. See #446.
+* [DEPENDENCY] bump psycopg2 to 2.7.1. See #295.
```

Comment:
Didn't this belong to the 5.13 release (on 2017-04-24) ? I guess you we should create a new section  1.0.2 and not reuse the last one.